README.md

```diff
@@ -55,13 +54,53 @@ We have some docs to make it easier to get you started:
 
 ## Deployment
 
-The website is built and deployed on [Netlify](https://netlify.com/) to our production environment ([https://yld.io](https://yld.io)) when:
+The website and lambda are built (`yarn build`) and deployed on [Netlify](https://netlify.com/) to our production environment ([https://yld.io](https://yld.io)) when:
 
 - a new commit is pushed to our **production** (`master`) branch, read more [here](https://www.netlify.com/docs/continuous-deployment/);
 - the Contentful data is updated (via _webhook_).
 
 Also, for each Pull Request that's open, a [Deploy Preview](https://www.netlify.com/blog/2016/07/20/introducing-deploy-previews-in-netlify/) is created, allowing for that branch to be tested and shared amongst stakeholders.
 
+## Automated deployments
+
+### 📬 Webhook automated deployments
+
+We are using Netlify webhooks/build hooks to automate new builds when services we use update. Each service has its own webhook url set up in Netlify.
+
+Keep in mind if you need to urgently alter any of the automated builds but don't have access to the services below, it's possible to just remove the webhook url from Netlify and stop it temporarily!
+
+#### Medium
+
+Zapier is subscribed to the yld engineering medium account via an RSS feeds, it checks every hour for new content. Upon new content zapier makes a post request to Netlify to trigger a fresh build.
```

Comment:
This is the first mention of Zapier anywhere in the docs. Might be good to just pop an brief explanation of what it is directly underneath the **## Automated deployments** heading, before we get started on webhooks/lambda sub-headings?